vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -137,12 +131,12 @@ ent prj pbs-publish
 ``` sh
 ent prj deploy
 ```
-See [this tutorial](../../tutorials/create/pb/publish-project-bundle.md) for more details.
+See [Build and Publish a Project Bundle](../../tutorials/create/pb/publish-project-bundle.md) for more details.
 
-### Install the bundle into an Application
-The CLI allows you to install a bundle without the need to access the Entando App Builder.
+### Install the Bundle into an Application
+The CLI allows you to install a bundle without accessing the Entando App Builder.
```

Comment:
accessing >> using ?

Reply:
i think "accessing" is appropriate here; the app builder is still used, but by the cli, not the user. the user doesn't have to interact with the app builder overtly but the app builder is still in play